admin.tf

```diff
@@ -0,0 +1,106 @@
+resource "aws_iam_role" "admin-ec2-role" {
```

Comment:
I don't really see the point of having those here publicly. Let's create them on AWS and then have them hardcoded here.